src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -623,4 +623,52 @@ public void executeQuery_incorrectQuery_doesNothing() {
 
     Assert.assertTrue(notificationId == 0);
   }
+
+  /** Tests function buildAnswerFull(). */
+  @Test
+  public void buildAnswer_validResultSet_successfulBuild() {
+    ResultSet resultSetMock = mock(ResultSet.class);
+    try {
+      // Mock behaviors for answer creation.
+      when(resultSetMock.getInt(SqlConstants.ANSWER_FETCH_ID)).thenReturn(1);
+      when(resultSetMock.getString(SqlConstants.ANSWER_FETCH_BODY)).thenReturn(
+          "You may get it at your college!");
+      when(resultSetMock.getString(SqlConstants.ANSWER_FETCH_AUTHORNAME)).thenReturn(
+          "Andres Abundis");
+      when(resultSetMock.getTimestamp(SqlConstants.ANSWER_FETCH_DATETIME)).thenReturn(
+          new Timestamp(1598899890));
```

Comment:
When you reference values multiple times in your test, it's probably best to make them into variables at the top of your test and then reference those variables throughout the test instead for the sake of readability!